starter-code/src/components/ContactsList.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import Contact from "./Contact";
+
+class ContactsList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      contacts: props.contacts.splice(0, 5),
+      sort: "none"
```

Comment:
This doesn't seem to be used.

Reply:
yeah, I thought I might need it for the deletion but then realized that it was rerendering the whole thing so each elem always already had its proper index